src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java

```diff
@@ -35,8 +35,24 @@ public static void main(String[] args) {
      * @return null/'RED'/'ORANGE'/'YELLOW'/'GREEN'
      */
     static String getColorHealthPoint(int currentHealthPoint, int maxHealthPoint) {
+        double healtPercent =  (double) (currentHealthPoint * 100) / maxHealthPoint;
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        if (healtPercent <= 0) {
+            return null;
+        }
+        if (healtPercent < 25){
+            return "RED";
+        }
+        else if (healtPercent < 50){
+            return "ORANGE";
+        }
+        else if (healtPercent < 75){
+            return "YELLOW";
+        }
+        else if (healtPercent <= 100){
+            return "GREEN";
+        }
+            return null;
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"